README.md

```diff
@@ -1,6 +1,9 @@
 blackbird
 =========
 
+[![Circle CI](https://circleci.com/gh/Vagrants/blackbird/tree/master.svg?style=shield)](https://circleci.com/gh/Vagrants/blackbird/tree/master)
+[![License](https://img.shields.io/badge/license-WTFPL-blue.svg)](http://www.wtfpl.net/about/)
+
```

Comment:
See https://github.com/Vagrants/blackbird/pull/44/files#diff-04c6e90faac2675aa89e2176d2eec7d8R130

Hmm... About License to write in two place.
I think that removing License at end of README, but I also think either is fine＼(^o^)／